rest_framework/schemas/openapi.py

```diff
@@ -98,9 +113,13 @@ def get_operation(self, path, method):
         if request_body:
             operation['requestBody'] = request_body
         operation['responses'] = self._get_responses(path, method)
+        operation['tags'] = self._get_tags(path, method)
 
         return operation
 
+    def get_tag_objects(self):
```

Comment:
I think drop this. (Not worth the complexity, which I think is the conclusion tom came to as well.)